### ISSUE TITLE
Running docker-compose in the background

### ISSUE THREAD

**Issue by edykes** [Senzing/docker-compose-demo#136]
We'd like to have the "docker-compose up" command run docker containers in the background, but what we are seeing is:

docker-compose up --detach --file resources/db2/docker-compose-rabbitmq-db2-again.yaml up       
Builds, (re)creates, starts, and attaches to containers for a service.

Unless they are already running, this command also starts any linked services.

The `docker-compose up` command aggregates the output of each container. When
the command exits, all containers are stopped. Running `docker-compose up -d`
starts the containers in the background and leaves them running.

If there are existing containers for a service, and the service's configuration
or image was changed after the container's creation, `docker-compose up` picks
up the changes by stopping and recreating the containers (preserving mounted
volumes). To prevent Compose from picking up changes, use the `--no-recreate`
flag.

If you want to force Compose to stop and recreate all containers, use the
`--force-recreate` flag.

Usage: up [options] [--scale SERVICE=NUM...] [--] [SERVICE...]

Options:
    -d, --detach               Detached mode: Run containers in the background,
                               print new container names. Incompatible with
                               --abort-on-container-exit.
    --no-color                 Produce monochrome output.
    --quiet-pull               Pull without printing progress information
    --no-deps                  Don't start linked services.
    --force-recreate           Recreate containers even if their configuration
                               and image haven't changed.
    --always-recreate-deps     Recreate dependent containers.
                               Incompatible with --no-recreate.
    --no-recreate              If containers already exist, don't recreate
                               them. Incompatible with --force-recreate and -V.
    --no-build                 Don't build an image, even if it's missing.
    --no-start                 Don't start the services after creating them.
    --build                    Build images before starting containers.
    --abort-on-container-exit  Stops all containers if any container was
                               stopped. Incompatible with -d.
    --attach-dependencies      Attach to dependent containers.
    -t, --timeout TIMEOUT      Use this timeout in seconds for container
                               shutdown when attached or when containers are
                               already running. (default: 10)
    -V, --renew-anon-volumes   Recreate anonymous volumes instead of retrieving
                               data from the previous containers.
    --remove-orphans           Remove containers for services not defined
                               in the Compose file.
    --exit-code-from SERVICE   Return the exit code of the selected service
                               container. Implies --abort-on-container-exit.
    --scale SERVICE=NUM        Scale SERVICE to NUM instances. Overrides the
                               `scale` setting in the Compose file if present.

**Comment by jeffadair** [comment]
It looks like this is the command you are running (please clarify if I am wrong)

> docker-compose up --detach --file resources/db2/docker-compose-rabbitmq-db2-again.yaml up

If that is the case, then there are to "up"s in the command, you only need the last one. Try

> docker-compose --detach --file resources/db2/docker-compose-rabbitmq-db2-again.yaml up

**Comment by docktermj** [comment]
@edykes Did the removal of the second `up` enable running in detached mode?

**Comment by docktermj** [comment]
@edykes  Let me know if your issue has been addressed.  If we don't hear back in 7 days, we'll go ahead and close the issue.

**Comment by docktermj** [comment]
Closing the issue. Feel free to reopen or create a new issue.
